index.html

```diff
@@ -5,22 +5,259 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Ben Omayio</title>
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="styles/main.css" />
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
+    <div class="desktop">
```

Comment:
@gtekle , thank you for the link on RWD. It seems i will have to delete the desktop version that i did and ammend the mobile version. This being optional changes (Not marked as required), i request to proceed to enable redo the changes latter to avoid repeating the block